### PR TITLE
Package cudajit.0.5.0

### PR DESCRIPTION
**Pull request by lukstafi**
### `cudajit.0.5.0`
Bindings to the `cuda` and `nvrtc` libraries with a unified interface
Bindings to manually selected parts of `lcuda` and `lnvrtc`, with a few types and conversion functions to facilitate use.

---
* Homepage: https://github.com/lukstafi/ocaml-cudajit
* Source repo: git+https://github.com/lukstafi/ocaml-cudajit.git
* Bug tracker: https://github.com/lukstafi/ocaml-cudajit/issues

---
:camel: Pull-request generated by opam-publish v2.3.1

### CODE REVIEW

**Comment by lukstafi** [comment]
Passing CI test target: https://github.com/lukstafi/ocaml-cudajit/actions/runs/11112452563/job/30874592644
I also updated lower bounds.

File: packages/cudajit/cudajit.0.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "Bindings to the `cuda` and `nvrtc` libraries with a unified interface"
+description:
+  "Bindings to manually selected parts of `lcuda` and `lnvrtc`, with a few types and conversion functions to facilitate use."
+maintainer: ["Lukasz Stafiniak"]
```
Comment (shonfeder) [inline packages/cudajit/cudajit.0.5.0/opam:7]: Could you add a contact email here? (This is part of a tighter policy we are trying to ease in.)

**Comment by lukstafi** [comment]
> See [#26542 (comment)](https://github.com/ocaml/opam-repository/pull/26542#issuecomment-2351692558) to explain the CI failures.
> 
> As an aside, and for the future, ye need a way to handle CI for this sort of package in the future, as it is not practical to add a `x-ci-accept-failure` for every platform. I'm opening a separate issue to track this.

Note the failures are already attributable to `conf-cuda`, which would be the primary beneficiary of https://github.com/ocurrent/opam-repo-ci/issues/371 .
So a separate feature request would be to have a more elaborate blame assignment. (I realize it's less trivial than just not blaming a package if its dependency failed to build.)

File: packages/cudajit/cudajit.0.5.0/opam (hunk 1 above)
Comment (lukstafi) [reply packages/cudajit/cudajit.0.5.0/opam:7]: Done (forgot about this).

**Comment by lukstafi** [comment]
Now also passing CI tests for:
- OCaml 4.12: https://github.com/lukstafi/ocaml-cudajit/actions/runs/11161753741/job/31024963647
- OCaml 5.2: https://github.com/lukstafi/ocaml-cudajit/actions/runs/11161753741/job/31024964021

**Comment by shonfeder** [comment]
Nice! Thanks for the additional CI results and for the fix here. Your [suggestion](https://github.com/ocaml/opam-repository/pull/26651#issuecomment-2391257810) is also excellent, and I've referred that upstream.

**Review by shonfeder — APPROVED** [review]

**Pull request merged**